addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -8,15 +8,51 @@ var widgetRegistry = require('web.widget_registry');
 // Display Notification alert on user preferences form view
 // -----------------------------------------------------------------------------
 var NotificationAlert = Widget.extend({
-   template: 'mail.NotificationAlert',
-   /**
-    * @override
-    */
-   init: function () {
-      this._super.apply(this, arguments);
-      var hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
-      this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
-   },
+    template: 'mail.NotificationAlert',
+    /**
+     * @override
+     */
+    init: function () {
+        this._super.apply(this, arguments);
+        var hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
+        this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
+        if (this.isNotificationBlocked) {
+            if (navigator.userAgent.search("Chrome") != -1) {
+                this.settingUrl = "chrome://settings/content/siteDetails?site=https%3A%2F%2Fdatabasename.odoo.com";
```

Comment:
this redirects to site details of `databasename.odoo.com`. Should use url in address bar.